search-app/src/cmr/search/site/data.clj

```diff
@@ -57,6 +95,7 @@
   "Given a list of provider maps, create the nested data structure needed
   for rendering providers in a template."
   [context tag providers]
+  (debug "Using providers:" providers)
```

Comment:
Is this debug still needed?

Reply:
It's for future debugging; when the log level is set to that, this is useful data that we'll want to check.